### PR TITLE
Feature/update graphs

### PR DESCRIPTION
**Pull request by LisaQ04**

### CODE REVIEW

File: waterlog/src/Components/JokeComponent.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { fetchNotifications } from '../actions/NumberOfNotifsActions';
+
+
+class Joke extends Component {
+  componentDidMount() {
+    this.timer = setInterval(()=>  this.props.fetchNotifications(), 5000);
+  }
+  componentWillUnmount() {
```
Comment (FredJEhlers) [inline waterlog/src/Components/JokeComponent.js]: Space between declarations

File: waterlog/src/Components/JokeComponent.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    clearInterval(this.timer)
+    this.timer = null;
+  }
+  render() {
+    console.log(this.props);
```
Comment (FredJEhlers) [inline waterlog/src/Components/JokeComponent.js]: left behind a console log

File: waterlog/src/ConfigRedux/enhancers/monitorReducer.js [hunk 3]
```diff
@@ -19,4 +19,4 @@ const monitorReducerEnhancer = createStore => (
     return createStore(monitoredReducer, initialState, enhancer)
 }
 
-export default monitorReducerEnhancer
+export default monitorReducerEnhancer
```
Comment (FredJEhlers) [inline waterlog/src/ConfigRedux/enhancers/monitorReducer.js]: empty line EOF and in `waterlog/src/ConfigRedux/middleware/logger.js` as well

File: waterlog/src/actions/NumberOfNotifsActions.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+export function fetchNotifications(){
+  return function(dispatch){
+    dispatch(fetchNotification());
+    return axios.get(process.env.REACT_APP_API_URL+'/api/realtime/pollnotifications', { headers: { 'Accept': 'application/json' }})
+    .then(function(result){
+      dispatch(fetchNotifSuccess(result))
+    })
+    .catch(error => dispatch(fetchNotifFail(error)));
+  }
+}
```
Comment (FredJEhlers) [inline waterlog/src/actions/NumberOfNotifsActions.js]: empty line EOF

File: waterlog/src/actions/Types.js [hunk 5]
```diff
@@ -96,6 +99,12 @@ export const FETCH_COSTS_FORECAST_FAILURE_MONTHLY = 'FETCH_COSTS_FORECAST_FAILUR
 export const FETCH_HEAT_MAP_DATA_BEGIN = 'FETCH_HEAT_MAP_DATA_BEGIN';
 export const FETCH_HEAT_MAP_DATA_SUCCESS = 'FETCH_HEAT_MAP_DATA_SUCCESS';
 export const FETCH_HEAT_MAP_DATA_FAILURE = 'FETCH_HEAT_MAP_DATA_FAILURE';
+//redux saga actions
+export const FETCH_NOTIF = 'FETCH_NOTIF';
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types.js:103]: Plan for your next sprint. Group these into a meaningful constant object so you guys don't have to individually import and export each line. 
Eg combine all map data in to one or all heat maps into one

File: waterlog/src/index.js [hunk 6]
```diff
@@ -17,4 +17,4 @@ const renderApp = () =>
     ) 
     
 renderApp()
-serviceWorker.register(); 
+serviceWorker.register();
```
Comment (FredJEhlers) [inline waterlog/src/index.js]: empty line EOF 
space between declarations

File: waterlog/src/reducers/NotifNumberReducer.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import { combineReducers } from 'redux';
+import { FETCH_NOTIF, FETCH_NOTIF_SUCCESS, FETCH_NOTIF_FAILURE } from '../actions/Types';
```
Comment (FredJEhlers) [inline waterlog/src/reducers/NotifNumberReducer.js]: Each import on its own line

File: waterlog/src/reducers/NotifNumberReducer.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+    default:
+      return state;
+  }
+};
+
+const rootReducer = combineReducers({
+  notif
+});
+
+export default rootReducer;
```
Comment (FredJEhlers) [inline waterlog/src/reducers/NotifNumberReducer.js]: Empty line EOF

File: waterlog/src/Components/JokeComponent.js (hunk 1 above)
Comment (LisaQ04) [reply waterlog/src/Components/JokeComponent.js]: Thanks this file was not used, I deleted it.

File: waterlog/src/Components/JokeComponent.js (hunk 2 above)
Comment (LisaQ04) [reply waterlog/src/Components/JokeComponent.js]: File was removed.

File: waterlog/src/ConfigRedux/enhancers/monitorReducer.js (hunk 3 above)
Comment (LisaQ04) [reply waterlog/src/ConfigRedux/enhancers/monitorReducer.js]: left empty line

File: waterlog/src/actions/NumberOfNotifsActions.js (hunk 4 above)
Comment (LisaQ04) [reply waterlog/src/actions/NumberOfNotifsActions.js]: left empty line EOF.

File: waterlog/src/actions/Types.js (hunk 5 above)
Comment (LisaQ04) [reply waterlog/src/actions/Types.js:103]: Thanks .

File: waterlog/src/index.js (hunk 6 above)
Comment (LisaQ04) [reply waterlog/src/index.js]: Fixed.

File: waterlog/src/reducers/NotifNumberReducer.js (hunk 7 above)
Comment (LisaQ04) [reply waterlog/src/reducers/NotifNumberReducer.js]: Formatted imports

File: waterlog/src/reducers/NotifNumberReducer.js (hunk 8 above)
Comment (LisaQ04) [reply waterlog/src/reducers/NotifNumberReducer.js]: Left empty line EOF.

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**